src/core_plugins/vega/public/vega_view/index.js

```diff
@@ -0,0 +1,3 @@
+export { VegaView } from './vega_view';
```

Comment:
imho, we don't really need this indirection. I'd just remove the file and import the dependencies directly

Reply:
@spalger wanted this :)  I'm fine either way.